lib/train/platforms/detect/specifications/os.rb

```diff
@@ -27,12 +27,27 @@ def self.load
       # unix master family
       plat.family('unix')
           .detect {
-            if unix_uname_s =~ /./
+            # we want to catch a special case here where cisco commands
+            # don't return an exit status and still print to stdout
+            if unix_uname_s =~ /./ && !unix_uname_s.start_with?('Line has invalid autocommand ')
               @platform[:arch] = unix_uname_m
               true
             end
           }
 
+      # cisco_ios family
+      plat.family('cisco').title('Cisco Family')
+          .detect {
+            true
+          }
+
+      plat.name('cisco_ios').title('Cisco IOS').in_family('cisco')
+          .detect {
+            return unless v == cisco_show_version
```

Comment:
I think you want this to be a single `=`, right? in-line assignment? Otherwise, I don't know where `v` is getting set and then it will either bomb here or in the next line when you try to fetch from it.

Reply:
ah darn linter caught me on that one, 1sec